### DISCUSSION TITLE
在自己国外的服务器上配置了docker服务但是现实无网络

### DISCUSSION THREAD

**Discussion by mlsqq** [1715173329/UnblockNeteaseMusic#1146] [Q&A]
```shell```
docker run -d \
--name wyy \
--restart=always \
-p 52136:8080 \
-e ENABLE_FLAC=true \
-e ENABLE_LOCAL_VIP=true \
-e JSON_LOG=true \
-e LOG_LEVEL=debug \
-e QQ_COOKIE="uin=; qm_keyst=" \
pan93412/unblock-netease-music-enhanced -o qq kuwo kugou migu 
```shell```
代理正常，但是不能解锁会员歌曲

**Comment by ghostvarg** [comment]
我在海外，是搭建在群晖Docker上得，是好用得，可以解锁歌曲，但是并不能解锁所有歌曲！
